extensions/trace-utils/src/main/java/io/opentelemetry/extension/trace/MessageEvent.java

```diff
@@ -21,8 +21,11 @@
  *
  * <p>It requires a {@link Type type} and a message id that serves to uniquely identify each
  * message. It can optionally have information about the message size.
+ *
+ * @deprecated This class will be removed in SDK 0.13.0.
  */
 @Immutable
+@Deprecated
 public final class MessageEvent {
```

Comment:
Add a note about the SemanticAttributes class maybe?

Reply:
Added one